From 4a48ba57e0cab63d357faf7adf93cf61106221f0 Mon Sep 17 00:00:00 2001
From: Alexander Grund <alexander.grund@tu-dresden.de>
Date: Mon, 7 Sep 2020 12:40:56 +0200
Subject: [PATCH] Remove pybind11 from required submodules

This can be taken from the system in which case it is not used from the
submodule. Hence the check here limits the usage unneccessarily
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 8c060a1c5e3f3..c91916ab1b1e8 100644
--- a/setup.py
+++ b/setup.py
@@ -309,7 +309,6 @@ def check_file(f):
             sys.exit(1)
 
     check_file(os.path.join(third_party_path, "gloo", "CMakeLists.txt"))
-    check_file(os.path.join(third_party_path, "pybind11", "CMakeLists.txt"))
     check_file(os.path.join(third_party_path, 'cpuinfo', 'CMakeLists.txt'))
     check_file(os.path.join(third_party_path, 'tbb', 'Makefile'))
     check_file(os.path.join(third_party_path, 'onnx', 'CMakeLists.txt'))
